website/integrations/services/nextcloud/index.md

```diff
@@ -98,13 +98,14 @@ To give authentik users admin access to your Nextcloud instance, you need to cre
 Create a custom SAML Property Mapping:
 
 -   Set the _SAML Attribute Name_ to `http://schemas.xmlsoap.org/claims/Group`.
--   Set the _Expression_ to:
+-   Set group names that you want to passthrough to start with 'NC-'
```

Comment:
@smdion what is the agreement on having this prefix? Sounds like @BeryJu and @rissson advise against it?